docs/content_management/pages/create_custom_page_block.md

```diff
@@ -132,5 +132,8 @@ The following UDW configuration is used so only an `event` typed content item ca
 
 For more information, see [UDW configuration](browser.md#udw-configuration).
 
-Your custom page block is now ready. 
-Before you can use it in Page Builder, you must [enable it in Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display).
+Your custom page block is now ready.
+
+!!! caution
+
+    Before you can use it in Page Builder, you must [enable it in Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display).
```

Comment:
May be we should link to the migration action?
```suggestion
    Before you can use the block in Page Builder, it needs to be enabled. This can be done manually in [Page field settings]([[= user_doc =]]/content_management/configure_ct_field_settings/#block-display) or by using the migration action [add_block_to_available_blocks](data_migration_actions/#available-migration-actions).
```